package.json

```diff
@@ -5,27 +5,51 @@
   "license": "MIT",
   "repository": "datproject/dat-desktop",
   "description": "Dat Desktop App",
-  "author": "Dat Team",
+  "author": "Dat Team <community@datproject.org>",
   "scripts": {
-    "start": "electron .",
+    "start": "cross-env NODE_ENV=development electron .",
     "pretest": "npm run lint",
     "test": "npm run test:unit",
-    "test:unit": "NODE_ENV=test babel-tape-runner -r ./unit-tests/_helpers/*.js unit-tests/*.js",
+    "test:unit": "cross-env NODE_ENV=test babel-tape-runner -r ./unit-tests/_helpers/*.js unit-tests/*.js",
     "lint": "prettier-standard 'app/**/*.js' 'tests/**/*.js' 'lib/**/*.js' 'unit-tests/*.js' && standard",
-    "build": "webpack",
-    "watch": "webpack --watch"
+    "deps": "cross-env DEBUG=* dependency-check . --detective precinct --entry app/index.js",
+    "clean": "git clean -dfX",
+    "burnthemall": "rm -rf ~/.electron && npm run clean && rm -f package-lock.json && npm install",
+    "update-rdt": "rm -rf dev/react-dev-tools && ced fmkadmapgofadopljbjfkapdkoienihi dev/react-dev-tools",
+    "dev": "cross-env NODE_ENV=development sh -c \"(npm run start &); npm run watch\"",
```

Comment:
shouldn't it be: `npm-run-parallel start watch`?

Reply:
It should, I just pushed the fixes (thanks for checking, seems like i forgot to save my edit)